packages/ui/components/pages/Fuse/FusePoolPage/SupplyList.tsx

```diff
@@ -413,6 +414,7 @@ const AssetSupplyRow = ({
                       <HStack mr={2}>
                         <Text fontSize={{ base: '3.2vw', sm: '0.9rem' }}>+</Text>
                         <TokenWithLabel address={info.rewardToken} size="2xs" />
+                        {!(info.formattedAPR && info.formattedAPR.gt(0)) && <ApyInformTooltip />}
```

Comment:
A bit tricky to understand the logic operation here.

Reply:
changed